packages/commonwealth/server/migrations/20240513183701-add-promotionalEmailsEnabled-to-profile.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
```

Comment:
Should this go in the `Profile` table? Seems like this would be better in the `SubscriptionPreferences` or even on the user. The profile email isn't the email used for sending emails I don't think.